catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Association.view.js

```diff
@@ -38,8 +38,21 @@ define([
             events: {
                 "click .remove-association": 'removeAssociation'
             },
+            initialize: function(options) {
+                this.readOnly = options.readOnly;
+            },
             removeAssociation: function () {
                 wreqr.vent.trigger('removeAssociation:' + this.model.get('sourceId'), this.model.get('id'));
+            },
+            serializeData: function() {
+                var data = {};
+
+                if (this.model) {
+                    data = this.model.toJSON();
+                }
+                data.readOnly = this.readOnly;
```

Comment:
It seems like the readOnly flag is always associated with a model that you're displaying. Is there a reason why you can't just set the flag on the model(s) and let it flow down that way instead of trying to mix in this value in all of these different views?